decidim-proposals/app/queries/decidim/proposals/metrics/proposal_followers_metric_measure.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Metrics
+      class ProposalFollowersMetricMeasure < Decidim::MetricMeasure
+        # Searches for unique Users following the next objects
+        #  - Proposals
+        #  - CollaborativeDrafts
+
+        def valid?
+          super && @resource.is_a?(Decidim::Component)
+        end
+
+        def calculate
```

Comment:
Split this method into different smaller methods, please

Reply:
Ok